python/setup.py

```diff
@@ -24,7 +24,7 @@
 # When the bazel version is updated, make sure to update it
 # in WORKSPACE file as well.
 
-SUPPORTED_BAZEL = (4, 2, 2)
+SUPPORTED_BAZEL = (5, 4, 0)
```

Comment:
This will require everyone to upgrade bazel. Just want to make sure we're ok with that (cc @rkooo567 @krfricke )

Reply:
Not aware of any particular issues